autovalet_gazebo/config/husky_gazebo_lot.rviz

```diff
@@ -344,6 +527,6 @@ Window Geometry:
     collapsed: false
   Views:
     collapsed: false
-  Width: 928
-  X: 65
+  Width: 936
+  X: 984
   Y: 24
```

Comment:
Why is this .rviz file being committed again?

Reply:
my bad.

Reply:
it's removed after rebasing. now good.